### ISSUE TITLE
Seeking guidance on 1.4.4 browser zoom, text size zoom etc

### ISSUE THREAD

**Issue by jamesjacobs** [w3c/wcag#2169]
After reading through 1.4.4 Resize text many times and also issues raised here relating to it I'm still not 100% convinced I've understood the requirements correctly. Hoping someone can illuminate me 😄 

Put simply, if a site displays correctly and doesn't cut off, clip, truncate or obscure text when the **page is zoomed in browser** (for example, by hitting CMD/CTRL + in Chrome to zoom the whole page) and doesn't prevent scaling or pinch and zoom on mobile - is that a pass? 

**Or** should a user be able to **zoom/increase only text** (for example, via methods such as FireFox text-only zoom or iOS Safari text zoom using the AA text sizing button)? If that results in text being cut off etc when a full page zoom displays correctly - is that a fail of 1.4.4?

Thanks in advance

**Comment by patrickhlauke** [comment]
the SC is silent about how users go about resizing text. in essence it demands that there be at least *one* way for users who need larger text to achieve this. no specifics about which methods a site must support. so yes, if the site works well with full-page zoom (even if it leads to bidirectional scrolling), then it's a pass - regardless of whether or not doing an explicit text-only resize on that same site would lead to problems. as long as one method works, it's a pass.

**Comment by fstrr** [comment]
You might find the note in [Failure of Success Criterion 1.4.4 when resizing visually rendered text up to 200 percent causes the text, image or controls to be clipped, truncated or obscured](https://www.w3.org/WAI/WCAG21/Techniques/failures/F69.html) useful:

> "The Working Group has discovered many misunderstandings about how to test this failure. We are planning to revise this failure in a future update. Until then, if the content passes the success criterion using any of the listed sufficient techniques, then it does not meet this failure."

**Comment by bruce-usab** [comment]
@jamesjacobs &mdash; please do reply back to this issue thread if you are satisfied with @patrickhlauke response (or explain what more you might be hoping for).  For better or worse, it is a pretty lightweight requirement.

**Comment by jamesjacobs** [comment]
Thanks for the confirmation all, that's really helpful. There seems to be so many conflicting opinions on this on the web, which resulted in our confusion.

Thanks again. Will close the ticket down.

**Comment by selfthinker** [comment]
> the SC [...] demands that there be at least _one_ way for users who need larger text to achieve this

The SC doesn't say that anywhere.

The Understanding page is, as usual, clear as mud. ;-) 
It specifically refers to "text" multiple times while also talking about browser zoom. I know, that is because WCAG is old and that text was written before browser zoom reliably existed in all browsers and before responsive web design became popular.
But especially when it talks about both it's clear to me that text (up to 200%) was deemed to be more important than zoom:

> The working group feels that 200% is a reasonable accommodation that can support a wide range of designs and layouts, and complements older screen magnifiers that provide a minimum magnification of 200%. Above 200%, zoom (which resizes text, images, and layout regions and creates a larger canvas that may require both horizontal and vertical scrolling) may be more effective than text resizing.

Furthermore, the intent is the most important part here:

> The intent of this Success Criterion is to ensure that visually rendered text [...] can be scaled successfully so that it can be read directly by people with mild visual disabilities, without requiring the use of assistive technology such as a screen magnifier. Users may benefit from scaling all content on the Web page, but text is most critical.

If someone uses text resize in Firefox and a website becomes inaccessible to them because important text is cut off, would you tell them they're just using "the wrong method" and they need to adapt to use browser zoom instead?

I would fail issues with text resize and not just browser zoom because it can make things quite inaccessible to real people.

**Comment by JAWS-test** [comment]
> If someone uses text resize in Firefox and a website becomes inaccessible to them because important text is cut off, would you tell them they're just using "the wrong method" and they need to adapt to use browser zoom instead?

Yes.

Understanding:

> A user uses a zoom function in his user agent to change the scale of the content. All the content scales uniformly, and the user agent provides scroll bars, if necessary.

https://www.w3.org/WAI/WCAG21/Techniques/failures/F94:

Use any of the following methods to resize text when available:

*  the zoom feature of the browser
*  the text-sizing feature of the browser,
*  on-page controls for resizing text.

Check that the text resizes by one of the methods above, and can be resized to at least 200% of the default. 

https://www.w3.org/WAI/WCAG21/Techniques/failures/F69:

The Working Group has discovered many misunderstandings about how to test this failure. We are planning to revise this failure in a future update. Until then, if the content passes the success criterion using any of the listed sufficient techniques, then it does not meet this failure.

**Comment by selfthinker** [comment]
> > If someone uses text resize in Firefox and a website becomes inaccessible to them because important text is cut off, would you tell them they're just using "the wrong method" and they need to adapt to use browser zoom instead?
> 
> Yes.

Harsh.
Does that mean that you, @JAWS-test, disagree with @patrickhlauke?

If something has overlapping and unreadable text when using browser zoom in Chrome but it works with text increase in Firefox, according to @patrickhlauke that would be a pass as there needs to be only "at least one way" to successfully make the text bigger. @JAWS-test, if I interpret what you wrote correctly, that would be a fail for you?

**Comment by KB55** [comment]
> The Working Group has discovered many misunderstandings about how to test this failure. We are planning to revise this failure in a future update. Until then, if the content passes the success criterion using any of the listed sufficient techniques, then it does not meet this failure.

I understand this text has been added but not into the main understanding document. It was my understanding that techniques/failures are informative and not normative in WCAG. Is there a reason why this hasn't been made part of the actual SC - and therefore clearing up the perceived misunderstanding? It is still there only as a note in WCAG 2.2 without being formally incorporated.

**Comment by bruce-usab** [comment]
> It was my understanding that techniques/failures are informative and not normative in WCAG.

That is correct.  They are also widely regarded as authoritative and the AGWG treats them seriously with all due deliberation.

> Is there a reason why this hasn't been made part of the actual SC?

Yes, because the phrasing of SC 1.4.4 is okay.  Also, with 2.1, I would note that we now have [SC 1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow) which goes further.

> It is still there only as a note [to F69] in WCAG 2.2 without being formally incorporated.

Yes, that is also correct.  While the AGWG has acknowledged that there are misunderstandings, we have not figured out what prose is needed to provide sufficient clarification.  Suggestions for edits to F69 (or to Understanding, or for new Techniques) are welcome!

**Comment by patrickhlauke** [comment]
as this question/topic seems to constantly bubble back up and cause confusion, I thought I'd take a stab at at least making the understanding document a tiny bit clearer (hopefully). Thoughts welcome https://github.com/w3c/wcag/pull/2270 /cc @alastc
